package.json

```diff
@@ -10,12 +10,15 @@
         "build": "tsc -b",
         "watch": "tsc --watch",
         "clean": "shx rm -rf lib",
-        "lint": "tslint --format stylish --project .",
-        "lint:fix": "tslint --format stylish --project . --fix",
-        "prettier": "prettier --write '**/*.{ts,tsx,json,md}' --config .prettierrc",
-        "prettier:ci": "prettier --list-different '**/*.{ts,tsx,json,md}' --config .prettierrc",
+        "tslint": "tslint --format stylish --project .",
+        "tslint:fix": "yarn tslint --fix",
+        "prettier": "prettier '**/*.{ts,tsx,json,md}' --config .prettierrc",
```

Comment:
I know we've had this convo before, but I personally would like to keep our `script` patterns the same as the monorepo.

If you feel strongly about grouping prettier & tslint into one "lint" command I see your point, but I do think it is a bit strange to have different patterns between this repo & the monorepo

Reply:
I appreciate your effort to keep us consistent.  FYI I'm a crusade to do this everywhere now.  Every time work in a monorepo package I'm updating it to include prettier in lint.  Though, so far I've only done it to asset-swapper and migrations :)

Reply:
🙀